packages/lesswrong/lib/analyticsEvents.js

```diff
@@ -99,6 +102,27 @@ export function useTracking({eventType="unnamed", eventProps = {}, captureOnMoun
 
 export const withTracking = hookToHoc(useTracking)
 
+export function useIsInView({rootMargin='0px', threshold=0}={}) {
+  const [entry, setEntry] = useState(false)
+  const ref = useRef()
+  useEffect(() => {
+    console.log({rootMargin, threshold}) // save for debuggin
```

Comment:
Lint

Reply:
The final PR, #2764 passes lint and has all console.log statements removed.